data/dapis/polygon-testnet.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "AVAX/USD": "0x981fb212c961973cb0e36c2ba003ab0d43e61f01035284e8c9917a296b78dbcb",
+  "ALPACA/USDT": "0x43442a225f83168907c82d86d69d0799160c0bf3fc221eb9e77bd3940da3205e"
+}
```

Comment:
Should we also add 'MATIC/USD'?

Reply:
Done :)
Now I'm waiting to see if CI passes :sweat_smile:

Reply:
It passed (!)
This is very cool :)